### ISSUE TITLE
Model not working?

### ISSUE THREAD

**Issue by darsh10** [andy194673/nlg-sclstm-multiwoz#3]
I see a dev/test slot error of 100.0. The train loss goes down. What exactly is going on?

**Comment by andy194673** [comment]
The model is working. Please refer to the updated training log and testing result.
Please note that if the initial learning rate is small (like 0.001), the slot error rate reduces slow. 
Change the initial learning rate to 0.005, you will see the model is working.

**Comment by darsh10** [comment]
Hi, thank you so very much for the response and for the bug fix. Yes, I did try a learning rate of 0.005. Yet I get a consistently 100% slot error rate. Can you share the pytorch version that you used, and the exact command used to create the log files ?

**Comment by darsh10** [comment]
Hi, I tried pytorch 0.3.1. However, the problem still persists. I don't get the warnings anymore. Yet the dev performance is 100% slot error rate.

**Comment by miroblog** [comment]
same issue, train/validation loss goes down ye,t, slot error rate is 100% . 
something is wrong with get_slot_error function.

**Comment by youngornever** [comment]
I have the same issue

**Comment by jin8** [comment]
After changing the code from `slot_tok = '@' + _das[0][:3] + '-' + _das[1] + '-' + _das[2]
` to `slot_tok = 'slot-'+_das.split(':')[1].lower()` , it solved the the problem of having slot error:100.000. 
 
![error](https://user-images.githubusercontent.com/1359313/70371869-c77eaa80-191b-11ea-98be-37c5b3c1dba4.JPG)
